concent_api/api-integration-force-accept-or-reject-results-test.py

```diff
@@ -5,35 +5,28 @@
 import random
 import time
 from base64                 import b64encode
+from collections import namedtuple
```

Comment:
Please follow the convention, `import namedtuple` should be aligned to other imports.